MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,236 @@
+description: Learn here all about the validation support in Syncfusion .NET MAUI DataForm (SfDataForm) and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Validation in .NET MAUI DataForm (SfDataForm)
+
+The data form validates the data and displays hints in the case of validation is not passed. In case of invalid data, the error message is shown at the bottom of the editor.
+
+## Built in validations
+
+The supported built in validations are as follows:
+
+* Data annotation
+
+### Data annotations
+
+You can validate the data using data annotation attributes.
+
+The numeric type like Int, Double, Decimal properties can be validated using the [Range](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.rangeattribute?view=netframework-4.8) attribute.
+
+{% tabs %}
+{% highlight c# %}
+
+    private int age;
+
+    [Range(1,120,ErrorMessage ="Age should be between 1 and 120")]
+    public int Age { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+The String type property can be validated using [Required](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=netframework-4.8) and [StringLength](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.stringlengthattribute?view=netframework-4.8) attributes.
+
+{% tabs %}
+{% highlight c# %}
+
+    private string name;
+
+    [Required(AllowEmptyStrings = false, ErrorMessage = "Name should not be empty")]
+    [StringLength(10, ErrorMessage = "Name should not exceed 15 characters")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Date range attribute
+
+You can validate the date time value using date range attribute.
+
+{% tabs %}
+{% highlight c# %}
+
+    private DateTime joinDate;
+
+    [DataType(DataType.Date)]
+    [DataFormDateRange(MinimumDate = "01/01/2022", MaximumDate = "31/12/2022", ErrorMessage = "Join date is invalid")]
+    public DateTime JoinDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validation mode
+
+The [ValidationMode] determines when the value should be validated.
+
+The supported validation modes are as follows:
+
+* LostFocus
+* PropertyChanged
+* Manual
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="ValidationMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm" 
+        ValidationMode="LostFocus">
+    </DataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.DataForm.ValidationMode = DataFormValidationMode.LostFocus;
+
+{% endhighlight %}
+{% endtabs %}
+
+### LostFocus
+
+If the commit mode is LostFocus, the value will be validated when the editor lost its focus.
```

Comment:
Mention lost focus is default validation mode

Reply:
Mentioned lost focus is default validation mode